raiden/channel/netting_channel.py

```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
 
                 raise InvalidLocksRoot(expected_locksroot, transfer.locksroot)
 
-            # For the sender: If the lock expires after the settle_period a
-            # secret could be revealed after the channel is settled and we
-            # won't be able to claim the token.
+            # For mediators: This is registering the the *mediator* paying
+            # transfer, the expiration of the lock must be `reveal_timeout`
+            # blocks smaller than the *received* paying transfer, this cannot
+            # be checked by the paying channel alone.
+            #
+            # For the initiators: There is no backing transfer, the expiration
```

Comment:
Can you please rephrase to: 
`For the initiators: As there is no backing transfer, the expiraton is arbitrary. Using the ...`